test.py

```diff
@@ -157,7 +185,11 @@ def target():
   commands = []
   dockerExtraArgs = []
   if os.environ.get("OPENMODELICAHOME"):
-    omc_cmd = ["%s/bin/omc" % os.environ.get("OPENMODELICAHOME")]
+    if isWin:
+      omc_cmd = ["%sbin\\omc" % os.environ.get("OPENMODELICAHOME")]
+    else:
+      omc_cmd = ["%s/bin/omc" % os.environ.get("OPENMODELICAHOME")]
```

Comment:
Does `OPENMODELICAHOME` path trailing slash existence or not differ between Windows and Linux? If so, that should maybe be fixed upstream? @AnHeuermann ?

Reply:
Use `os.path.join` to unify both cases. It will deal with extra slashes and use the appropriate `/` or `\\`.